### PR TITLE
INFRA-195

### PR DESCRIPTION
**Pull request by sterlingwellscaffeine**
1.) Added ability to read keyfiles from JSON object in additional to being able to read from the file path.

2.) Added unit testing for both of these use cases.

### CODE REVIEW

File: src/modules/KeyManager.js [hunk 1]
```diff
@@ -190,160 +150,222 @@ class KeyManager {
...
 
     /* ------------------------------ Public methods -------------------------------- */
+
     /**
+       * Imports the keyfile data object
+       * @param {object} keyStorage: The JS object representing the encrypted keyfile
+       * @param {object} password: The password to unlock the keyfile
+       * @returns {object} returns the keyStorage used in the KeyManager
+       */
+    importKeyPair(keyStorage, password) {
```
Comment (scasplte2) [inline src/modules/KeyManager.js:160]: Should these be static methods to generate a new keyManager? This way KeyManager begins to resemble PrivateKeyCurve25519 more cloesly

File: src/modules/KeyManager.js [hunk 2]
```diff
@@ -190,160 +150,222 @@ class KeyManager {
...
+      }
+    };
+
+      /**
+         * Imports key data object from keystore JSON file.
+         * @param {string} filepath the filepath of the keystore JSON
+         * @param {string} password the password for encrypting/decrypting * the keyfile
+         * @returns {object} returns the keyStorage used in the KeyManager
+         */
+      importFromFile = (filepath, password) => {
```
Comment (scasplte2) [inline src/modules/KeyManager.js:192]: similar to above, might want to make this static

File: src/modules/KeyManager.js (hunk 1 above)
Comment (sterlingwellscaffeine) [reply src/modules/KeyManager.js]: Updated with new changes.

File: src/modules/KeyManager.js (hunk 2 above)
Comment (sterlingwellscaffeine) [reply src/modules/KeyManager.js]: Updated with new changes.

File: src/modules/KeyManager.js [hunk 3]
```diff
@@ -202,6 +241,24 @@ class KeyManager {
...
 
+    /**
+       * Setter function to input keyStorage in the Bifrost compatible format
+       * @param {object} keyStorage - The keyStorage object that the keyManager will use to store the keys for a particular address.
+       * @param {string} password for encrypting (decrypting) the keyfile
+       * @function setKeyStorage
+       * @memberof KeyManager
+       * @returns {object};
+       */
+    setKeyStorage(keyStorage, password) {
```
Comment (raularagonez) [inline src/modules/KeyManager.js:252]: Let's make this private.

**Review by raularagonez — CHANGES_REQUESTED** [review]
I see the constructor is messy, needs to be revisited for readability/maintainability purposes.

File: src/modules/KeyManager.js (hunk 3 above)
Comment (raularagonez) [reply src/modules/KeyManager.js]: I made the changes to make it private.

**Review by raularagonez — APPROVED** [review]
👍🏽  Merging, let's discuss KeyManager module for future iterations.

**Pull request merged**